### ISSUE TITLE
Uncaught Error: ENOENT: no such file or directory, stat '/Users/po/tmp/tmp/link'

### ISSUE THREAD

**Issue by negipo** [hmatsuda/file-explorer#10]
[Enter steps to reproduce below:]

```
$ mkdir tmp
$ cd tmp
$ ln -s doesnotexists link
$ touch file
$ atom file
```

then press `cmd-alt-n` to reproduce.
It may be caused by bronken symlink.

**Atom Version**: 1.0.0
**System**: Mac OS X 10.10.3
**Thrown From**: [file-explorer](https://github.com/hmatsuda/file-explorer) package, v0.2.3

### Stack Trace

Uncaught Error: ENOENT: no such file or directory, stat '/Users/po/tmp/tmp/link'

```
At fs.js:846

Error: ENOENT: no such file or directory, stat '/Users/po/tmp/tmp/link'
    at Error (native)
    at fs.statSync (fs.js:846:18)
    at Object.fs.statSync (ATOM_SHELL_ASAR.js:206:16)
    at FileExplorerView.module.exports.FileExplorerView.viewForItem (/Users/po/.atom/packages/file-explorer/lib/file-explorer-view.coffee:40:17)
    at FileExplorerView.module.exports.SelectListView.populateList (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:206:29)
    at FileExplorerView.module.exports.SelectListView.setItems (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:167:12)
    at FileExplorerView.module.exports.FileExplorerView.populate (/Users/po/.atom/packages/file-explorer/lib/file-explorer-view.coffee:160:19)
    at FileExplorerView.module.exports.FileExplorerView.toggle (/Users/po/.atom/packages/file-explorer/lib/file-explorer-view.coffee:116:14)
    at FileExplorerView.module.exports.FileExplorerView.toggleCurrentDirectory (/Users/po/.atom/packages/file-explorer/lib/file-explorer-view.coffee:102:21)
    at atom-text-editor.<anonymous> (/Users/po/.atom/packages/file-explorer/lib/file-explorer.coffee:31:49)
```

### Commands

```
     -0:00.8.0 file-explorer:toggle-current-directory (atom-text-editor.editor.vim-mode.command-mode.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ],
    "ignoredNames": [
      ".git",
      ".svn",
      ".DS_Store"
    ],
    "disabledPackages": [
      "recent-files",
      "cute-cursor",
      "highlight-column",
      "git-grep",
      "git-grep",
      "git-grep",
      "dash",
      "recent-files-fuzzy-finder",
      "rails-finder",
      "minimap",
      "minimap-git-diff"
    ],
    "projectHome": "/Users/po/src/github.com/negipo",
    "excludeVcsIgnoredPaths": false
  },
  "file-explorer": {
    "excludeActiveFile": false
  }
}
```

### Installed Packages

```coffee
# User
auto-update-packages, v1.0.0
autocomplete-paths, v1.0.2
autocomplete-ruby, v0.1.0
color-picker, v2.0.7
css-snippets, v0.8.0
editor-stats, v0.17.0
emmet, v2.3.12
file-explorer, v0.2.3
file-icons, v1.5.8
highlight-line, v0.11.0
highlight-selected, v0.10.1
japanese-wrap, v0.2.7
language-gitignore, v0.2.0
language-haml, v0.21.0
language-rspec, v0.3.0
open-recent, v2.2.4
project-manager, v1.15.10
rspec-snippets, v0.4.0
trailing-spaces, v0.3.2
vim-mode, v0.54.0
vim-mode-visual-block, v0.2.7
vim-surround, v0.7.2

# Dev
No dev packages
```

**Comment by hmatsuda** [comment]
Thanks for reporting. I published v-0.2.4 with this fixing.
https://atom.io/packages/file-explorer

**Comment by negipo** [comment]
:heart_eyes: :+1:
